providers/azureblob/src/main/java/org/jclouds/azure/storage/filters/SharedKeyLiteAuthentication.java

```diff
@@ -281,4 +284,19 @@ void appendUriPath(HttpRequest request, StringBuilder toSign) {
       }
    }
 
+   private String join(String[] parts, String delimiter) {
```

Comment:
Can you use the Guava `Joiner` instead?

Reply:
Sure, sorry, didn't notice guava is already there.